auto_py_to_exe/web/js/i18n.js

```diff
@@ -5,205 +5,238 @@ const translationMap = {
-                ru: 'Добавить шаблон'
+                ru: 'Добавить шаблон',
+                de: 'Leerzeichen hinzufügen'
             },
             importConfig: {
                 en: 'Import Config From JSON File',
                 zh: '从JSON文件导入配置',
                 zh_tw: '從 JSON 檔案導入配置',
-                ru: 'Импортировать Конфигурацию из JSON Файла'
+                ru: 'Импортировать Конфигурацию из JSON Файла',
+                de: 'Konfiguration aus JSON-Datei importieren'
             },
             exportConfig: {
                 en: 'Export Config To JSON File',
                 zh: '将配置导出到JSON文件',
                 zh_tw: '將配置導出到 JSON 檔案',
-                ru: 'Экспортировать Конфигурацию в JSON Файл'
+                ru: 'Экспортировать Конфигурацию в JSON Файл',
+                de: 'Konfiguration in JSON-Datei exportieren'
             },
             convert: {
                 en: 'Convert .py to .exe',
                 zh: '将.PY转换为.EXE',
                 zh_tw: '將.PY 轉換為 .EXE',
-                ru: 'Конвертировать .py В .exe'
+                ru: 'Конвертировать .py В .exe',
+                de: 'Konvertiere .py in .exe'
             },
             openOutputFolder: {
                 en: 'Open Output Folder',
                 zh: '打开输出目录',
                 zh_tw: '打開輸出目錄',
-                ru: 'Открыть Папку Вывода'
+                ru: 'Открыть Папку Вывода',
+                de: 'Ausgabeverzeichnis öffnen'
             },
             enable: {
                 en: 'Enable',
                 zh: '开启',
                 zh_tw: '開啟',
-                ru: 'Включить'
+                ru: 'Включить',
+                de: 'Aktivieren'
             },
         },
         links: {
             helpPost: {
                 en: 'Help Post',
                 zh: '帮助帖子',
                 zh_tw: '幫助文章',
-                ru: 'Справка'
+                ru: 'Справка',
+                de: 'Hilfe-Post'
             }
         },
         placeholders: {
             pathToFile: {
                 en: 'Path to file',
                 zh: '文件路径',
                 zh_tw: '檔案路徑',
-                ru: 'Путь к файлу'
+                ru: 'Путь к файлу',
+                de: 'Pfad zur Datei'
             },
             icoFile: {
                 en: '.ico file',
                 zh: '图标路径',
                 zh_tw: '圖示路徑',
-                ru: '.ico файл'
+                ru: '.ico файл',
+                de: '.ico Datei'
             },
             directory: {
                 en: 'DIRECTORY',
                 zh: '目录',
                 zh_tw: '目錄',
-                ru: 'Директория'
+                ru: 'Директория',
+                de: 'VERZEICHNIS'
             },
             arguments: {
                 en: 'ARGUMENTS',
                 zh: '参数',
                 zh_tw: '參數',
-                ru: 'Аргументы'
+                ru: 'Аргументы',
+                de: 'ARGUMENTE'
             }
         },
         helpText: {

```

Comment:
Missing help text translations - we can still merge without these though as they simply won't be replaced